Dockerfile.arm32v7

```diff
@@ -17,15 +17,19 @@
 FROM arm32v7/python:3.7.6-buster
 
 # Upgrade all packages to meet security criteria
-RUN apt-get update && apt-get upgrade -y && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get upgrade -y && apt-get install sudo && rm -rf /var/lib/apt/lists/*
 
 # arm32v7 is the CPU that is integrated into Raspberry Pi 4
 
-WORKDIR /workspace
-RUN mkdir assets
-
 # Enable piwheels
 RUN echo "[global]\nextra-index-url=https://www.piwheels.org/simple" >> /etc/pip.conf
 
+
+RUN useradd --create-home max && echo "max ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+USER max
+WORKDIR /home/max
+RUN mkdir assets
+
 COPY . .
 RUN pip install --upgrade pip && pip install -r requirements.txt
```

Comment:
A note here, the CI prints:

> Defaulting to user installation because normal site-packages is not writeable

This would be different from the intel one, which uses miniconda and site-packages is writable

Reply:
Do you think we should run this line with `sudo` to install in the root directory?

Reply:
actually if you want uniformity, it might be the easiest to replace the intel base image with `python:3.7.6-buster`. Then we can sync them up almost completely.